swagger.yml

```diff
@@ -475,10 +475,14 @@ definitions:
     type: object
     x-nullable: false
     properties:
-      upgrade:
+      allowReboot:
```

Comment:
a `description` field for the two fields would show up in the documentation and maybe safe as a few support inquires. 
In both cases its unclear what these fields actually do.

Reply:
thanks. added